communication/nats/sender.go

```diff
@@ -18,10 +18,12 @@
 package nats
 
 import (
-	"github.com/mysterium/node/communication"
 	"time"
 
+	"github.com/mysterium/node/communication"
+
 	"fmt"
+
```

Comment:
The order of imports here looks strange :/ Standart ones - our custom - standart again?

Reply:
Fixed. When I accidently add to many newlines, goimports dont reformat my "better style"